mitmproxy/optmanager.py

```diff
@@ -92,7 +92,7 @@ class OptManager:
         mutation doesn't change the option state inadvertently.
     """
     def __init__(self):
-        self.deferred: typing.Dict[str, typing.List[str]] = {}
+        self.deferred: typing.Dict[str, typing.Any] = {}
```

Comment:
If I understand it correctly, value of the option in this dictionary could be of any allowed type for options.